library/ajax/drug_autocomplete/search.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * weno rx search.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+require_once('../../../interface/globals.php');
+
+
+if (isset($_GET['term'])) {
+    $return_arr = array();
+    $term    = filter_input(INPUT_GET, "term");
+    
+    try {
+        $sql = "SELECT drug_label_name, price_per_unit FROM erx_drug_paid WHERE drug_label_name LIKE ? ";
+        $val = array('%'.$term.'%');
+        $res = sqlstatement($sql, $val);
+        while ($row = sqlFetchArray($res)) {
+            $return_arr[] =  $row['drug_label_name'] . " - ". $row['price_per_unit'];
+        }
+    } catch (PDOException $e) {
```

Comment:
This seems odd since the database connector is not even PDO???

Reply:
left over from the original code that was PDO.

Reply:
then would completely remove this try/catch (note sqlStatement outputs errors already)
as an aside, does these even work since the S in sqlstatement is not capitalized??

Reply:
Done